src/data/ACTIONS/root/SGE.ts

```diff
@@ -1,5 +1,298 @@
+	/** Single-Target Heals/Shields */
+	DIAGNOSIS: {
+		id: SGE_STUB + 1,
+		name: 'Diagnosis',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 1500,
+		mpCost: 400,
+	},
+	EUKRASIAN_DIAGNOSIS: {
+		id: SGE_STUB + 2,
+		name: 'Eukrasian Diagnosis',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1500,
+		mpCost: 1000,
+		statusesApplied: ['EUKRASIAN_DIAGNOSIS', 'DIFFERENTIAL_DIAGNOSIS'],
+	},
+	HAIMA: {
+		id: SGE_STUB + 3,
+		name: 'Haima',
+		icon: '',
+		cooldown: 120000,
+		statusesApplied: ['HAIMATINON'],
+	},
+	EGEIRO: {
+		id: SGE_STUB + 4,
+		name: 'Egeiro',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 8000,
+		mpCost: 2400,
+	},
+
+	/** AoE Heals/Shield */
+	PROGNOSIS: {
+		id: SGE_STUB + 5,
+		name: 'Prognosis',
+		icon: '',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 2000,
+		mpCost: 1000,
+	},
+	EUKRASIAN_PROGNOSIS: {
+		id: SGE_STUB + 6,
+		name: 'Eukrasian Prognosis',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1500,
+		mpCost: 1100,
+		statusesApplied: ['EUKRASIAN_PROGNOSIS'],
+	},
+	PANHAIMA: {
+		id: SGE_STUB + 7,
+		name: 'Panhaima',
+		icon: '',
+		cooldown: 120000,
+		statusesApplied: ['PANHAIMATINON'],
+	},
+	PHYSIS: {
+		id: SGE_STUB + 8,
+		name: 'Physis',
+		icon: '',
+		cooldown: 60000,
+		statusesApplied: ['PHYSIS'],
+	},
+	PHYSIS_II: {
+		id: SGE_STUB + 9,
+		name: 'Physis II',
+		icon: '',
+		cooldown: 180000,
+		statusesApplied: ['PHYSIS_II'],
+	},
+	HOLOS: {
+		id: SGE_STUB + 10,
+		name: 'Holos',
+		icon: '',
+		cooldown: 60000,
+	},
+	PEPSIS: {
+		id: SGE_STUB + 11,
+		name: 'Pepsis',
+		icon: '',
+		cooldown: 15000,
+	},
+
+	/** Utility */
+	KARDIA: {
+		id: SGE_STUB + 12,
+		name: 'Kardia',
+		icon: '',
+		cooldown: 5000,
+	},
+	EUKRASIA: { // TODO: Apparently this is on the GCD? Will need confirmation after launch
+		id: SGE_STUB + 13,
+		name: 'Eukrasia',
+		icon: '',
+		onGcd: true,
+		gcdRecast: 1000,
+	},
+	SOTERIA: {
+		id: SGE_STUB + 14,
+		name: 'Kardia',
+		icon: '',
+		cooldown: 90000,
+		statusesApplied: ['SOTERIA'],
+	},
+	ICARUS: {
+		id: SGE_STUB + 15,
+		name: 'Icarus',
+		icon: '',
+		cooldown: 45000,
+	},
+	ZOE: {
+		id: SGE_STUB + 16,
+		name: 'Zoe',
+		icon: '',
+		cooldown: 90000,
+		statusesApplied: ['ZOE'],
+	},
+	KRASIS: {
+		id: SGE_STUB + 17,
+		name: 'Krasis',
+		icon: '',
+		cooldown: 60000,
+		statusesApplied: ['KRASIS'],
+	},
+
+	/** Addersgall abilities */
+	RIZOMATA: {
+		id: SGE_STUB + 18,
+		name: 'Rizomata',
+		icon: '',
+		cooldown: 90000,
+	},
+	DRUOCHOLE: {
+		id: SGE_STUB + 19,
+		name: 'Druochole',
+		icon: '',
+		cooldown: 1000,
+	},
+	IXOCHOLE: {
+		id: SGE_STUB + 20,
+		name: 'Ixochole',
+		icon: '',
+		cooldown: 30000,
+	},
+	KERACHOLE: {
+		id: SGE_STUB + 21,
+		name: 'Kerachole',
+		icon: '',
+		cooldown: 30000,
+		statusesApplied: ['KERACHOLE', 'KERACHOLE_REGEN'],
+	},
+	TAUROCHOLE: {
+		id: SGE_STUB + 22,
+		name: 'Taurochole',
+		icon: '',
+		cooldown: 30000,
```

Comment:
Cooldown should be `45000` for this one.

Reply:
Sure 'nuff